poppy/instrument.py

```diff
@@ -927,13 +950,13 @@ def _get_weights(self, source=None, nlambda=None, monochromatic=None, verbose=Fa
             # Allow user to provide directly a tuple, as in poppy.calc_psf source option #3
             return source
 
-        else:  # Fallback simple code for if we don't have pysynphot.
+        else:  # Fallback simple code for if we don't have stsynphot.
             poppy_core._log.warning(
-                "Pysynphot unavailable (or invalid source supplied)!   Assuming flat # of counts versus wavelength.")
+                "stsynphot unavailable (or invalid source supplied)! Assuming flat # of counts versus wavelength.")
             # compute a source spectrum weighted by the desired filter curves.
-            # The existing FITS files all have wavelength in ANGSTROMS since that is the pysynphot convention...
+            # The existing FITS files all have wavelength in ANGSTROMS since that is the stsynphot convention...
             filterfile = self._filters[self.filter].filename
-            filterfits = fits.open(filterfile)
+            filterfits = fits.open(filterfile)  # TODO: Use context manager
```

Comment:
This #TODO could be done now simply by replacing this line and the next with `filterdate = fits.getdata(filterfile)`, I think.